src/avt/Queries/Queries/avtHohlraumFluxQuery.C

```diff
@@ -604,9 +609,16 @@ avtHohlraumFluxQuery::IntegrateLine(int oneSide, int otherSide,
         pt0[1]  = pt1[1];
         pt0[2]  = pt1[2];
     }
-    for (ii = 0 ; ii < numBins ; ii++)
+    if (currPt == endPt)
+    {
+        for (ii = 0 ; ii < numBins ; ii++)
+        {
+            radBins[ii] += tmpBins[ii];
+        }
+    }
+    else
     {
-        radBins[ii] += tmpBins[ii];
+        debug5 << "Throwing away line." << endl;
```

Comment:
You might want to add information to the debug message that this is coming from the HolraumFluxQuery.